src/ray/core_worker/task_execution/actor_scheduling_queue.cc

```diff
@@ -250,6 +258,32 @@ void ActorSchedulingQueue::ScheduleRequests() {
   }
 }
 
+void ActorSchedulingQueue::CancelAllPendingUnsafe(const Status &status) {
+  // Cancel in-order pending tasks
+  while (!pending_actor_tasks_.empty()) {
+    auto head = pending_actor_tasks_.begin();
+    RAY_LOG(INFO) << "Canceling pending in-order actor task due to shutdown: seq_no="
```

Comment:
Please ignore the verbose logging for now. I added as I have been debugging a flaky test. Will remove some of them before merging.